lib/rules/no-dataset.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(context) {
+  return {
+    "Identifier": function(node) {
```

Comment:
Ok, actually I _really_ did it in https://github.com/github/eslint-plugin-github/pull/13/commits/77a89e0fac80933f565f28097c4521e963f7a3ec.